pages/candy-machines/[candy_machine_ID].tsx

```diff
@@ -77,7 +76,7 @@ const CandyMachine: NextPage = () => {
             setError('')
             try {
                 setIsLoading(true)
-                const provider = new AnchorProvider(connection, anchorWallet, {
+                const provider = new AnchorProvider(new Connection(connection.rpcEndpoint), anchorWallet, {
                     preflightCommitment: 'processed',
```

Comment:
Better to use `confirmed` instead of `processed`, this way the account propagation will most likely be favorable.

Reply:
In this PR im only merging `dev` into `main`, im not implementing anything, if we can improve the code around this, better create an issue in github and implement the enhancements in a different PR 🙏🏼